static/components/TaskDetails.jsx

```diff
@@ -250,10 +246,10 @@ var TaskDetails = React.createClass({
           ref: task.sha,
         },
         success: (data) => {
-          let {app, environment, number} = this.props.task;
-          browserHistory.push(`/deploys/${app.name}/${environment}/${number}`);
-        }
-      });
+          browserHistory.push('/');
+          browserHistory.push(`/deploys/${data.app.name}/${data.environment}/${data.number}`);
```

Comment:
Solution was loosely based off of work arounds founds here: https://github.com/ReactTraining/react-router/issues/1982